bindings/wasm/identity_wasm/examples/src/util.ts

```diff
@@ -1,139 +1,89 @@
-    address: Address;
-    document: IotaDocument;
-    fragment: string;
-}> {
-    const didClient = new IotaIdentityClient(client);
-    const networkHrp: string = await didClient.getNetworkHrp();
-
-    const secretManagerInstance = new SecretManager(secretManager);
-    const walletAddressBech32 = (await secretManagerInstance.generateEd25519Addresses({
-        accountIndex: 0,
-        range: {
-            start: 0,
-            end: 1,
-        },
-        bech32Hrp: networkHrp,
-    }))[0];
-
-    console.log("Wallet address Bech32:", walletAddressBech32);
-
-    await ensureAddressHasFunds(client, walletAddressBech32);
-
-    const address: Address = Utils.parseBech32Address(walletAddressBech32);
+import { IotaClient } from "@iota/iota-sdk/client";
+import { getFaucetHost, requestIotaFromFaucetV0 } from "@iota/iota-sdk/faucet";
+
+export const IDENTITY_IOTA_PACKAGE_ID =
+    process.env.IDENTITY_IOTA_PACKAGE_ID || "0xac854096fcbfadcdd8cc8e4b6242d1b35607ef5324bfe54ba7a4be69fa6db36d";
+export const NETWORK_NAME_FAUCET = "localnet";
+export const NETWORK_URL =
+    process.env.NETWORK_URL || "http://127.0.0.1:9000";
+export const TEST_GAS_BUDGET = BigInt(50_000_000);
+
+export function getMemstorage(): Storage {
+    return new Storage(new JwkMemStore(), new KeyIdMemStore());
+}
 
+export async function createDocumentForNetwork(storage: Storage, network: string): Promise<[IotaDocument, string]> {
     // Create a new DID document with a placeholder DID.
-    // The DID will be derived from the Alias Id of the Alias Output after publishing.
-    const document = new IotaDocument(networkHrp);
+    const unpublished = new IotaDocument(network);
 
-    const fragment = await document.generateMethod(
+    const verificationMethodFragment = await unpublished.generateMethod(
         storage,
         JwkMemStore.ed25519KeyType(),
         JwsAlgorithm.EdDSA,
-        "#jwk",
-        MethodScope.AssertionMethod(),
+        "#key-1",
+        MethodScope.VerificationMethod(),
     );
 
-    // Construct an Alias Output containing the DID document, with the wallet address
-    // set as both the state controller and governor.
-    const aliasOutput: AliasOutput = await didClient.newDidOutput(address, document);
-
-    // Publish the Alias Output and get the published DID document.
-    const published = await didClient.publishDidOutput(secretManager, aliasOutput);
-
-    return { address, document: published, fragment };
+    return [unpublished, verificationMethodFragment];
 }
 
-/** Request funds from the faucet API, if needed, and wait for them to show in the wallet. */
-export async function ensureAddressHasFunds(client: Client, addressBech32: string) {
-    let balance = await getAddressBalance(client, addressBech32);
-    if (balance > BigInt(0)) {
-        return;
+export async function getClientAndCreateAccount(storage: Storage): Promise<IdentityClient> {
```

Comment:
maybe just `getClient` or `getFundedClient`?

Reply:
Hm, makes sense, picked the latter one, and aligned that helper functions siblings a bit:

- `bindings/wasm/identity_wasm/examples/src/util.ts` (the one from above)
  `getClientAndCreateAccount` -> `getFundedClient`
- `examples/utils/utils.rs` (same role, just in Rust)
  `get_client_and_create_account` -> `get_funded_client`
- `identity_iota_core/tests/e2e/common.rs` (similar role, but returns a test specific instance)
  `get_client` -> `get_funded_test_client`